src/transformers/processing_utils.py

```diff
@@ -557,14 +569,21 @@ def get_processor_dict(
         is_local = os.path.isdir(pretrained_model_name_or_path)
         if os.path.isdir(pretrained_model_name_or_path):
             processor_file = os.path.join(pretrained_model_name_or_path, PROCESSOR_NAME)
+            chat_template_file = os.path.join(pretrained_model_name_or_path, "chat_template.json")
+
         if os.path.isfile(pretrained_model_name_or_path):
             resolved_processor_file = pretrained_model_name_or_path
+            # cant't load chat-template when given a file as pretrained_model_name_or_path
+            resolved_chat_template_file = None
```

Comment:
I'm not sure I understand this. If `pretrained_model_name_or_path` is a file, couldn't we check if there's a chat template file under the same path ?

Reply:
My understanding was that if `pretrained_model_name_or_path` is a file, its a `processor_config.json` file. I might be wrong, as my usual way to load model is from the hub

Reply:
ah, right, sorry, you're right